bdd/step-definitions/e2e/cli.ts

```diff
@@ -203,6 +203,7 @@ Then("I get instance health", { timeout: 10000 }, async function() {
 
 Then("health outputs 404", async () => {
     stdio = await getStreamsFromSpawn("/usr/bin/env", [...si, "inst", "health", instanceId, ...connectionFlags()]);
+    console.log(stdio);
```

Comment:
:D 
I must admit I asked @patuwwy to leave it, because after blood draining fight with the BDD tests on CI, finally this round the tests passed. I have had a similar issue with @jan-warchol and the Python PRs today. I confirm the fails are pretty "random". It seems our BDD tests on CI has become unreliable. We should take a look at what is happening.